iOS/samples/sdksample/BingMapsSdkSample/ViewController.swift

```diff
@@ -109,9 +112,19 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
         }
 
         setupDemoMenu()
+
+        currentStyle = MapStyle.roadLight
         updateMapViewColorScheme()
```

Comment:
If I'm following the proposed changes correctly, the behavior will be such that:
- For the *SDK sample*, the app ends up overriding the map style sheet selection made by the end user when the `userInterfaceStyle` trait changes.
- For the *Search sample*, the app always adjusts the map style sheet based on the current `userInterfaceStyle`.
The latter makes sense to me since the canvas plays a more passive role in that sample; but I think the behavior on the former should be the other way around:
- On launch pick a sensible default based on the default UI style, i.e. `UIScreen.main.traitCollection.userInterfaceStyle` or inherited `userInterfaceStyle`.
- Afterwards let the user's selection drive the UI style, i.e. set `overrideUserInterfaceStyle`.

Reply:
Agreed.